src/main/scripts/jbang.cmd

```diff
@@ -82,18 +82,21 @@ SETLOCAL DISABLEDELAYEDEXPANSION
 set ERROR=%ERRORLEVEL%
 rem catch errorlevel straight after; rem or FOR /F swallow would have swallowed the errorlevel
 
-if %ERROR% NEQ 0 (
-  del ""%tmpfile%""
+if %ERROR% EQU 0 (
+  rem read generated java command by jang, delete temporary file and execute.
+  for %%A in ("%tmpfile%") do for /f "usebackq delims=" %%B in (%%A) do (
+    set "OUTPUT=%%B"
+    goto :break
+  )
+:break
+  del "%tmpfile%"
+  %OUTPUT%
+  exit /b %ERRORLEVEL%
+) else if %ERROR% EQU 3 (
+  type "%tmpfile%"
+  del "%tmpfile%"
+  exit /b 0
```

Comment:
same as above - consider if we can just execute on error 0 and anything else we print AND return that exit code (not just 0)